src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -376,3 +379,27 @@ export function decodeScalar(
       : undefined;
   return isString(unwrapped) ? unwrapped : undefined;
 }
+
+export function downloadAsCsv(tableData, filename) {
```

Comment:
@wmak As discussed, we can pass `eventView` from here https://github.com/getsentry/sentry/blob/eb18bea23be662c9a2c08caedabd410bc535f742/src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx#L375 to this function.

Then from there we can generate the column order ` const columnOrder = eventView.getColumns();`. This would provide the column order instead of the columns from the `meta`.

You can generate the appropriate keys using this function https://github.com/getsentry/sentry/blob/eb18bea23be662c9a2c08caedabd410bc535f742/src/sentry/static/sentry/app/views/eventsV2/utils.tsx#L247-L256, which would allow you to map the data from `tableData`.